team.daangn.com/src/templates/jobs.tsx

```diff
@@ -9,6 +9,7 @@ import { required } from '@cometjs/core';
 
 import PageTitle from '~/components/PageTitle';
 import _JobPostList from '~/components/JobPostList';
+import expandMoreOutlineUrl from '~/assets/expand_more_outline_m.svg'
```

Comment:
외부 이미지로 넣는거 너무 좋아요. 근데 이거 내용이 작아서 url-loader가 사용되는 거 같은데 file-loader로 강제하는게 더 좋을 것 같아요. critical path에 포함할 만한 요소는 아니라서요

```suggestion
import expandMoreOutlineUrl from '!!file-loader!~/assets/expand_more_outline_m.svg'
```

Reply:
머 지금도 이미지가 다 들어가는건 아니긴 하네요. JS랑 CSS에만 중복해서 들어감

Reply:
아 svg작아서 신경안썼는데 file-loader 쓰는게 좋긴하겠네요.